### PR TITLE
Automated cherry pick of #2182: Hide anomaly detection when FIPS is enabled

### PR DESCRIPTION
**Pull request by rene-dekker**
Cherry pick of #2182 on release-v1.28.

#2182: Hide anomaly detection when FIPS is enabled

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Review by Brian-McM — APPROVED** [review]

**Pull request merged**
